### ISSUE TITLE
CSS Custom Properties - resolve not working

### ISSUE THREAD

**Issue by ryc76** [tomblachut/svelte-intellij#95]
Hi

It appears CSS Custom Properties are not supported. For example if you create a custom property 

```javascript
<style type="text/css">
:root {
      --field-border-top: #ebebe0 1px solid;
}
.all-borders {
      border-radius: var(--field-border-top);
}
</style>
```
Webstorm will display a error stating "Cannot resolve --field-border-top custom property.

Thanks

**Comment by tomblachut** [comment]
Thanks for reporting. I don't know yet why it isn't working.

**Comment by tomblachut** [comment]
Fix released as part of 0.17.*
